.circleci/config.yml

```diff
@@ -70,17 +70,17 @@ jobs:
   build_for_tests:
     executor: main-executor
     environment:
-      - FEDERATION: "true"
-      - NODE_ENV: "test"
-      - ACL_SERVICE_URL: "https://acl-intb.ciscospark.com/acl/api/v1"
-      - ATLAS_SERVICE_URL: "https://atlas-intb.ciscospark.com/admin/api/v1"
-      - ENCRYPTION_SERVICE_URL: "https://encryption-intb.ciscospark.com/encryption/api/v1"
-      - IDENTITY_BASE_URL: "https://identitybts.webex.com"
-      - CONVERSATION_SERVICE: "https://conversation-intb.ciscospark.com/conversation/api/v1"
-      - HYDRA_SERVICE_URL: "https://apialpha.ciscospark.com/v1/"
-      - IDBROKER_BASE_URL: "https://idbrokerbts.webex.com"
-      - U2C_SERVICE_URL: "https://u2c-intb.ciscospark.com/u2c/api/v1"
-      - WDM_SERVICE_URL: "https://wdm-intb.ciscospark.com/wdm/api/v1"
+      FEDERATION: true
+      NODE_ENV: test
+      ACL_SERVICE_URL: https://acl-intb.ciscospark.com/acl/api/v1
+      ATLAS_SERVICE_URL: https://atlas-intb.ciscospark.com/admin/api/v1
+      ENCRYPTION_SERVICE_URL: https://encryption-intb.ciscospark.com/encryption/api/v1
+      IDENTITY_BASE_URL: https://identitybts.webex.com
+      CONVERSATION_SERVICE: https://conversation-intb.ciscospark.com/conversation/api/v1
+      HYDRA_SERVICE_URL: https://apialpha.ciscospark.com/v1/
+      IDBROKER_BASE_URL: https://idbrokerbts.webex.com
+      U2C_SERVICE_URL: https://u2c-intb.ciscospark.com/u2c/api/v1
+      WDM_SERVICE_URL: https://wdm-intb.ciscospark.com/wdm/api/v1
```

Comment:
I like this code style change, much more consistent with the rest of the content.